content/docs/start/visualization.md

```diff
@@ -0,0 +1,101 @@
+# Visualization with DVC Plots
+
+As we discussed in [experiments] section, DVC versions the experiments by
+associating parameters and metrics to them. On top of that, DVC can generate
+plots and tracks plot images you generated from the code. As in other aspects of
+the experiments, DVC is flexible and provides more than one option to fit
+visualization to your workflow:
+
+- DVC can generate HTML files that includes interactive [Vega-Lite] plots, from
+  the data series in JSON, YAML, CSV, or TSV.
+
+- DVC can keep track of image files produced as plot outputs from the
+  training/evaluation scripts.
+
+- DVC, with its [DVCLive] integration can produce plots automatically during
```

Comment:
DVCLive can do all 3 of these plot types now 🎉 , but I agree with your focus here on the "live monitoring" functionality that distinguishes it from the other plots. Should we make it more clear that this is about tracking progress of metrics updates in realtime?